client/lib/abtest/active-tests.js

```diff
@@ -224,14 +224,6 @@ export default {
 		allowExistingUsers: false,
 		countryCodeTargets: [ 'US', 'CA' ],
 	},
-	verticalsOnGutenboarding: {
-		datestamp: '20200811',
-		variations: {
-			verticals: 50,
-			noverticals: 50,
-		},
-		defaultVariation: 'noverticals',
-	},
```

Comment:
Together with removing the test definition, we can probably revert also its usage: https://github.com/Automattic/wp-calypso/pull/44256/files#diff-46ce4cd8580489aebbb3447a87c3cb57